src/components/ReportUpload/CounterUploadModal/CounterUpload.js

```diff
@@ -132,17 +148,24 @@ class CounterUpload extends React.Component {
     });
   };
 
+  cancleUpload = () => {
+    this.closeInfoModal();
+    this.setState({
+      selectedFile: {},
```

Comment:
I think it also needed to clear the state of `reportEditedManually` and `editReason`.

I tried to cancel the upload with given `reportEditedManually` and `editReason`. After trying to upload a report again, the `editReason` specified before cancel was used but not shown.

Reply:
Thanks for the feedback! Did not realized that behaviour.